e2e/cluster.go

```diff
@@ -21,7 +21,7 @@ import (
 // this will avoid potential conflicts with tests running on other branches
 // there is no strict rules or a hidden meaning for the version
 // testClusterNamePrefix is also used for versioning cluster configurations
-const testClusterNamePrefix = "abe2e-v20240725-"
+const testClusterNamePrefix = "abe2e-v20240801-"
```

Comment:
when would you say one should update this? generally when making large behavioral changes?

Reply:
When you make any change to the cluster config.

I reduced system nodepool size. I think it's not allowed by create or update AKS cluster operation.

Reply:
maybe worth adding a comment to clarify?

Reply:
I tried. Do you have suggestions to improve it?

Reply:
all clusters should be automatically deleted after 2-4 days